core/src/main/java/io/onetable/client/TableFormatClientFactory.java

```diff
@@ -39,13 +38,13 @@ public static TableFormatClientFactory getInstance() {
   }
 
   public TargetClient createForFormat(
-      TableFormat tableFormat, PerTableConfig perTableConfig, Configuration configuration) {
+      String tableFormat, PerTableConfig perTableConfig, Configuration configuration) {
     switch (tableFormat) {
-      case ICEBERG:
+      case "ICEBERG":
```

Comment:
Should use use the constants that you defined in the `TableFormat` class?

Reply:
done.